### ISSUE TITLE
⚠ Artifacts download error on Bintray ⚠

### ISSUE THREAD

**Issue by dxnxk-0** [hmrc/emailaddress#29]
I got SBT errors and cannot build my app anymore, and I suspect all apps using this lib won't build neither. The download request on Bintray results in a 403 forbidden response. Can you fix the repository ASAP ?

**Comment by dxnxk-0** [comment]
For the record, I have just removed my dependency to this lib. If this library is not maintained anymore, it would be wise to clearly state it in the README.

**Comment by ElkhdadiOussama** [comment]
@pascaldanek yes I'm facing the same problem right thanks for declaring it, and btw can you tell me which alternative you are using right now

**Comment by dxnxk-0** [comment]
Hi @ElkhdadiOussama i have basically integrated the code and i dont use any library anymore.
